VueJS/index.js

```diff
@@ -1,13 +1,27 @@
-import List from './components/List.js'
-import ListItem from './components/ListItem.js'
+import TodoList from './components/List.js'
+import TodoItem from './components/ListItem.js'
 
-new Vue({
+var app = new Vue({
   el: '#app',
-  data: {
-    todos: [
-      { id: '1', message: 'Foo', },
-      { id: '2', message: 'Bar', },
-      { id: '3', message: 'Baz', }
-    ]
+  data() {
+    return {
+      newTodoText: '',
+      todos: [
+        { id: '1', message: 'Einkaufen', },
+        { id: '2', message: 'Aufräumen', },
+        { id: '3', message: 'Putzen', }
+      ],
+      nextTodoId: 4
```

Comment:
Same suggestion as in the other review here: https://github.com/Xnasav/Systems-Development-and-Frameworks/pull/2#pullrequestreview-307893862

A developer would always have to remember to increment this when you want to change the list `todos`. Why not a randomly generated `id`? The keys only need to be unique, they don't need to be an ascending number sequence.